lib/node_modules/@stdlib/blas/base/sscal/lib/ndarray.js

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2019 The Stdlib Authors.
+* Copyright (c) 2020 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2024 The Stdlib Authors.
```

I am not sure where the `2020` changes are coming from.